pkg/gds/config/config.go

```diff
@@ -21,7 +22,7 @@ type Config struct {
 	Maintenance bool                `split_words:"true" default:"false"`
 	LogLevel    logger.LevelDecoder `split_words:"true" default:"info"`
 	ConsoleLog  bool                `split_words:"true" default:"false"`
-	GDS         GDSConfig
+	API         GDSConfig
```

Comment:
This is to avoid weird environment variables such as `GDS_GDS_`, luckily I didn't have to update many other places.

Reply:
Makes sense, the other two environment variables, `Enabled` and `BindAddr` had directly specified environment variables, but this is probably more extensible.